src/Hamiltonians/GutzwillerSampling.jl

```diff
@@ -21,16 +21,28 @@ After construction, we can access the underlying Hamiltonian with `G.hamiltonian
 
 ```jldoctest
 julia> H = HubbardMom1D(BoseFS{3}((1,1,1)); u=6.0, t=1.0)
```

Comment:
```suggestion
julia> H = HubbardMom1D(BoseFS((1,1,1)); u=6.0, t=1.0)
```
Hmm, it looks like something went wrong here with automated fix of a doctest. This needs revisiting.